tests/integration_tests/test_async_download.py

```diff
@@ -0,0 +1,92 @@
+import io
+from datetime import datetime
+from unittest.mock import Mock
+
+import pytest
+from notifications_python_client.notifications import NotificationsAPIClient
+
+from config import Config
+from core.aws import _S3_CLIENT
+from core.const import DATETIME_ISO_8601, EXCEL_MIMETYPE
+from core.controllers.async_download import get_file_format_from_extension, get_human_readable_file_size
+
+
+@pytest.fixture()
+def uploaded_exel_mock_file(seeded_test_client):
+    """Uploads a mock Excel file and deletes it on tear down."""
+    fake_file = io.BytesIO(b"0x01010101")
+    key = datetime.now().strftime(DATETIME_ISO_8601)
+    metadata = {"filename": "fake_file.xlsx"}
+    _S3_CLIENT.put_object(
+        fake_file,
+        Config.AWS_S3_BUCKET_FIND_DATA_FILES,
+        key,
+        ExtraArgs={"Metadata": metadata, "ContentType": EXCEL_MIMETYPE},
+    )
+    yield
+    _S3_CLIENT.delete_object(Bucket=Config.AWS_S3_BUCKET_FIND_DATA_FILES, Key=key)
+
+
+@pytest.fixture()
+def uploaded_json_mock_file(seeded_test_client, test_buckets):
+    """Uploads a mock JSON file and deletes it on tear down."""
+    fake_file = io.BytesIO(b"0x01010101")
+    key = f"fund-monitoring-data-{datetime.now().strftime(DATETIME_ISO_8601)}"
+    metadata = {"filename": "fake_file.json"}
+    _S3_CLIENT.put_object(
+        fake_file,
+        Config.AWS_S3_BUCKET_FIND_DATA_FILES,
+        key,
+        ExtraArgs={"Metadata": metadata, "ContentType": "application/json"},
+    )
+    yield
+    _S3_CLIENT.delete_object(Bucket=Config.AWS_S3_BUCKET_FIND_DATA_FILES, Key=key)
+
+
+@pytest.fixture()
+def app_context(test_session):
+    with test_session.application.app_context:
+        yield
+
+
+@pytest.fixture()
+def mocked_notify_success():
+    notifications_client = Mock(spec=NotificationsAPIClient)
+    notifications_client.send_email_notification.return_value = (
+        "success",
+        200,
+    )
```

Comment:
I don't think any of this is used any more?

Reply:
Removed these test bits.